README.md

```diff
@@ -13,10 +13,18 @@ The Http instance will run the WebRequest coroutines for you so you dont have to
 ## Requirements
 Unity 2017.3 and above (Required for GetTexture, SendWebRequest() and Assembly Definitions).
 
+## Installation
+It's recommended that you submodule this repo to your Assets directory using:  
+`git submodule add git@github.com:dubit/unity-http.git Assets/DUCK/Http`  
```

Comment:
Now it's modular, should we still be encouraging people to group them all within one `/DUCK/` folder? It might make more sense as a standalone thing in `Assets/Vendor/Http` for example. I'm not sure how the package manager does it's filepaths though, if you're just keeping parity with that.

Reply:
It's just a recommendation, they can put it wherever they want, the most important part that this line plays is that this repo is not a Unity Project, so it needs to go inside Assets. From this line they can make the decision of what path they want and easily adjust it (Catering for all levels of knowledge of git). The line just shows the intention and my preference ;)